_posts/release/2014-10-03-1.6.1.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Apache Accumulo 1.6.1 Release Notes
+title: Apache Accumulo 1.6.1 released
```

Comment:
I think we can drop the word "released" from these titles. They aren't really adding much value.